### PR TITLE
Migrate from src to source

### PR DESCRIPTION
**Pull request by bryophyta**
## What and why?

Upgrade from old versions of the Guardian Source packages (`src-*`) to the new versions (`source-*`). The old versions are deprecated, so this PR reduces technical debt by making sure that we're using supported dependencies moving forwards.

Most of the changes are just updating and consolidating imports, but I've also moved the code away from deprecated features of Source (e.g. preferring `palette` over `background`).

## Images

>**Note**
>The screenshots look like they have different vertical spacing, but I think this is just because github is displaying the screenshots in the second column as a slightly smaller width. Comparing the two in browser, I can't see any points where they don't line up.

|Before|After|
|-|-|
|||
|---|---|
|image|image|

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -196,20 +198,26 @@ const HomePage = (): jsx.JSX.Element => (
...
+          <h3 css={[readerFundedHeadingCss, printReaderFundedHeadingCss]}>
+            Print Subscriptions
+          </h3>
           <div css={readerFundedSubscribeCardHolderCss}>
             <ReaderFundedSubscribeCard
               title="Newspaper"
               bodyText="Convenient and money-saving, get a newspaper delivered to your door, or pick it up from your local shop. Choose your subscription, from daily to weekend-only."
               href="https://support.theguardian.com/subscribe/paper"
-              imagePath="/about/images/newspaper-desktop.png" />
+              imagePath="/about/images/newspaper-desktop.png"
```
Comment (georgeblahblah) [inline src/pages/index.tsx:213]: something for another time, but I noticed these images are huge (720kb, 1844x520px) for the size they're displayed at

**Review by georgeblahblah — APPROVED** [review]
Thanks @bryophyta! I had one comment for another time about the size of some of the static image assets. I think they're bigger than they need to be, given their usage.

Additionally, I noticed some changes which look like formatting fixes, but I don't think we have an explicit pretter config for this repo. It might be worth adding one as a separate change? But maybe I missed it!

**Comment by bryophyta** [comment]
>Additionally, I noticed some changes which look like formatting fixes, but I don't think we have an explicit pretter config for this repo. It might be worth adding one as a separate change? But maybe I missed it!

Thanks for this! Yep I left them in because it looked like they were adding consistency rather than changing things at random (and because I didn't want to fight the editor 😅) but I was also thinking we should add a config here. Do you think that we could just re-use the DCR config?

>something for another time, but I noticed these images are huge (720kb, 1844x520px) for the size they're displayed at

Good catch! I'll open an issue 👍

**Comment by georgeblahblah** [comment]
> Do you think that we could just re-use the DCR config?

Yes I think so! It looks like DCR's config just points to [`@guardian/pretter`](https://github.com/guardian/csnx/tree/main/libs/%40guardian/prettier), which is probably the right choice for this project

**Pull request merged**